### PR TITLE
Templates

### PR DESCRIPTION
**Pull request by ThierryO**
Add a README with installation instructions and templates for INBO report, INBO slides and Flemish government slides. This pull request solves issues #6 and #7. 

Can @inbo/bmk, @peterdesmet  and @stijnvanhoey have a look at this? Place comments in this pull request. Or fix problems in the templates branche ;-)

### CODE REVIEW

**Comment by ThierryO** [comment]
FYI @inbo/bmk, @peterdesmet and @stijnvanhoey : If you want to test this in your R version, you need to install the `INBOmd` version in the `templates` branch. Note that you also need the latest version of `INBOtheme`.

    # install.packages("devtools")
    devtools::install_github("inbo/INBOtheme")
    devtools::install_github("inbo/INBOmd", ref = "templates")

**Comment by ivyjansen** [comment]
Nog een interessante optie voor in _bookdown.yml : "clear_book: TRUE"
Dat lost de cache problemen op.

**Comment by ThierryO** [comment]
@ivyjansen that will actually remove the cache after building the book. So caching is pointless with `clearbook: TRUE`

**Comment by florisvdh** [comment]
Some comments that relate to the current state of the `templates` branch, regarding the report templates:

I wondered what was the motivation behind:

- replacing `pandoc.table()` and `xtable()` by `kable()`. I would rather add them all in the elaborate template, and mention their respective (dis)advantages.
- setting `language: english` in the template, and translating the templates in English.

Anyway, my commits (#18) are in line with those decisions. The `pandoc.table()` can still be re-added, though.

Further remarks:

- under subsection codesize, you should indicate what 'text' versus 'maximum' mean in the table.
- the background colours of the blocks are too dark for easy reading.

**Comment by florisvdh** [comment]
I want to propose making the 'single-Rmd-approach' versus the 'multiple-Rmd-approach' the _only_ fundamental distinction between both report templates, as far as RMarkdown code is concerned. Especially, I see no more need for two different ways of cross-referencing; the bookdown approach is not more complicated. I would use the bookdown-derived output formats in both templates, as they also work on single Rmd files. Then, one is less restricted to one output format when using cross-referencing syntax.

In my pull-request (#18), I did not do such changes though, apart from elaborating alternative output formats. Such changes would have effect on the simple report template, where cross-referencing syntax is now for pdf-output only.

Note: the bookdown output formats also seem to compile with rmarkdown::render(), for single Rmd files.

**Comment by ThierryO** [comment]
- kable() instead of pandoc.table(): the nice thing about kable() is that is know what the output format of the document is and it will automatically switch to the optimal formating language. This yields nicer tables.
- the main language of the package is English. Therefore I wrote the templates in English too.
- maybe we should use only the bookdown template and indicate that the extra Rmd files are optional (but recommended in case of a larger document).

**Comment by florisvdh** [comment]
* `kable()`: fair enough :-), after all it is up to the user to add more stuff. Still, I thought `pandoc.table()` had its advantages and supported more complexity, like specifying column width and such. I tend to use `kable()` for simpler tables, and `pandoc.table()` for more complex ones.
* language: OK
* I think it would be a good choice indeed, to deprecate the simple template and add clarification on single/multiple Rmd in the bookdown one. Having just one report template will also simplify further development of the template.

**Comment by florisvdh** [comment]
In the report that I'm making with the bookdown version of inbo_rapport_2015, I see that page numbers are at the left side of pages for new parts and chapters. Shouldn't these pages be at the right side in a report, and therefore their pagenumbers as well?

File: inst/local_tex/tex/latex/inbogeneric_2015/inbocolours_2015.sty [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+\definecolor{alert.colour.title}{RGB}{255, 255, 255}
+\definecolor{alert.colour.body}{RGB}{228, 56, 145}
+\definecolor{alert.colour}{RGB}{132, 56, 96}
+\definecolor{example.colour.title}{RGB}{255, 255, 255}
+\definecolor{example.colour.body}{RGB}{68, 196, 110}
+\definecolor{example.colour}{RGB}{45, 125, 71}
+\definecolor{warning.colour}{RGB}{192, 67, 132}
+\definecolor{highlight.colour}{RGB}{192, 67, 132}
+\definecolor{link.colour}{RGB}{53, 97, 150}
+\definecolor{cite.colour}{RGB}{58, 163, 92}
```
Comment (florisvdh) [inline inst/local_tex/tex/latex/inbogeneric_2015/inbocolours_2015.sty]: @ThierryO Shouldn't we copy the values of `highlight.colour` to `link.colour` and `cite.colour`?

**Pull request merged**